api/WebKitCSSMatrix.json

```diff
@@ -21,8 +21,7 @@
           },
           "ie": [
             {
-              "version_added": "11",
-              "prefix": "WebKit"
+              "version_added": "11"
             },
             {
               "version_added": "10",
```

Comment:
On the line below this one, we should change it to `alternative_name: "MSCSSMatrix"` instead.

Reply:
> On the line below this one, we should change it to `alternative_name: "MSCSSMatrix"` instead.

OK, added that